docs/Microservices/RPC/buildRPC_Manual.md

```diff
@@ -0,0 +1,228 @@
+In addition to using GitHub Actions to obtain a binary, the RPC binary can also be manually built. The steps below walk through how to build RPC on Windows 10, Ubuntu (18.04 or 20.04), and CentOS7/8.
+
+### Required Software
+
+- [git](https://git-scm.com/downloads)
+
+Additionally, if using Windows 10:
+
+- [Microsoft Visual Studio*](https://visualstudio.microsoft.com/): 2019 or newer version of Visual Studio Community/Professional
+    - Make sure to install the **Desktop development with C++** package at time of installation or via the 'Get tools and extensions' menu within Microsoft Visual Studio*.
+
+<br>
+
+The steps below assume the following directory structure where rpc is the clone of the [rpc repository](https://github.com/open-amt-cloud-toolkit/rpc), vcpkg is a clone of the VCPKG tool source and build is the RPC build directory. Both vcpkg and build directories will be created in later steps.
+
+```
+\rpc
+  |__vcpkg
+  |__build
+```
+
+
+### Clone the Repository
+
+1. On your development system, navigate to a directory of your choice to clone and build RPC.
+
+2. Clone the RPC repository.
+
+    ``` bash
+    git clone https://github.com/open-amt-cloud-toolkit/rpc.git && cd rpc
+    ```
+
+### Install Prerequisites and Build RPC
+
+=== "Windows"
+    Open 'x64 Native Tools Command Prompt for VS 20XX' as Administrator on your development system.  **This is NOT a regular Windows Command Prompt.**  This specific tool is used for compiling the RPC executable.
+    
```

Comment:
Administrator now required for building. Only for running.

Reply:
I'm fine with the increased privilege if it makes it easier for the user (think about it now, get it over with, as opposed to later). Two different terminals, one with Admin Privileges, one non-Admin, is that gonna just get people confused. Anyway.